pkg/registry/resource/deviceclass/strategy.go

```diff
@@ -89,21 +87,6 @@ func (deviceClassStrategy) AllowUnconditionalUpdate() bool {
 	return true
 }
 
-// dropDisabledFields removes fields which are covered by the optional DRAControlPlaneController feature gate.
+// dropDisabledFields removes fields which are covered by a feature gate.
 func dropDisabledFields(newClass, oldClass *resource.DeviceClass) {
-	if utilfeature.DefaultFeatureGate.Enabled(features.DRAControlPlaneController) {
-		// No need to drop anything.
-		return
-	}
-
-	if oldClass == nil {
-		// Always drop on create.
-		newClass.Spec.SuitableNodes = nil
-		return
-	}
-
-	// Drop on update only if not already set.
-	if oldClass.Spec.SuitableNodes == nil {
-		newClass.Spec.SuitableNodes = nil
-	}
 }
```

Comment:
This function is now empty... Are you leaving it around because you know a future PR will end up populating it again?

Reply:
Yes.